test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/003_ATF_N_TC_new_navi_app_certificate_exist.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] SDL must start PTU for navi app right after app successfully registration
+-- [PolicyTableUpdate] New application has registered and doesn't yet exist in Local PT during PTU in progress
+--
+-- Description:
+-- In case navigation app connects and sucessfully registers on SDL (opens RPC 7 service)
+-- and PolicyTable has NO "certificate" at "module_config" section of LocalPolicyTable
+-- SDL must start PolicyTableUpdate process on sending SDL.OnStatusUpdate(UPDATE_NEEDED) to HMI to get "certificate"
+--
+-- 1. Used preconditions:
+-- Navi app doesn't exist in LP, certificate is listed in module_config
+--
+-- 2. Performed steps
+-- Register navi application.
+--
+-- Expected result:
+-- Application is registered successfully.
+-- SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
```

Comment:
@dboltovskyi, based on clarification "Clarification for PTU triggers: navi app(no certificate) and app registered not listed in PT." with this script is checked requirement: " [PolicyTableUpdate] New application has registered and doesn't yet exist in Local PT during PTU in progress" that is not affected because of new CRQ. 
PTU will be triggered based on new application registered.